src/service/caldav.py

```diff
@@ -232,7 +232,7 @@ def incomplete_task(self, user_id, task_list, task_id):
             try:
                 calendar = self.principals[user_id].calendar(cal_id=task_list)
                 todo = calendar.object_by_uid(task_id, comp_class=Todo)
-                todo.uncomplete()
+                todo.incomplete()
```

Comment:
This is supposed to be `uncomplete`, it's from another library, I didn't write it.